kpi/utils/mongo_helper.py

```diff
@@ -260,7 +260,7 @@ def _get_cursor_and_count(cls, mongo_userform_id, hide_deleted=True,
             if ObjectId.is_valid(query.get('_uuid')):
                 query['_uuid'] = ObjectId(query.get('_uuid'))
             else:
-                raise ValidationError(_('Invalid _uuid specified'))
+                raise ValidationError({'_uuid': _('Invalid')})
```

Comment:
would the field in this case be `q`? ideally the validation wouldn't be this deep, but then we'd have to move database-specific code up into the validator so it would know what a good uuid looks like :/